project/SiteConfig.scala

```diff
@@ -54,6 +54,10 @@ object SiteConfig {
         "Streaming",
         "http4s is built on FS2, a streaming library that provides for processing and emitting large payloads in constant space and implementing websockets.",
       ),
+      Teaser(
+        "Cross-platform",
+        "http4s cross-builds for Scala.js, enabling you to share code across browser and electron clients, serverless functions, and JVM servers."
```

Comment:
electron clients? 
Maybe something like

browser, nodejs, electron, serverless functions and JVM servers

Reply:
Thanks, that does read funny. Maybe drop electron altogether, do people know what that is?